NLog.Web.ASPNET5/LayoutRenderers/AspNetRequestValueLayoutRenderer.cs

```diff
@@ -137,4 +137,19 @@ protected override void DoAppend(StringBuilder builder, LogEventInfo logEvent)
             }
         }
     }
+
+    internal static class RequestAccessor
+    {
+        internal static HttpRequestBase TryGetRequest(this HttpContextBase context)
+        {
+            try
+            {
+                return context.Request;
+            }
+            catch (HttpException)
+            {
+                return null;
```

Comment:
The compiler refuses to accept `InternalLogger.Debug(ex)`. It will only accept two variants of the Debug method: One taking a string, and the other taking a string and a argument array. Is that as excepted?